ios/Runner/AppDelegate.swift

```diff
@@ -75,6 +80,40 @@ import Flutter
             }
         })
         
+        let unstoppableDomainChannel = FlutterMethodChannel(name: "com.cakewallet.cake_wallet/unstoppable-domain", binaryMessenger: controller.binaryMessenger)
+        unstoppableDomainChannel.setMethodCallHandler({ [weak self]
+                            (call: FlutterMethodCall, result: @escaping FlutterResult) -> Void in
+                            switch call.method {
+                            case "getUnstoppableDomainAddress":
+                                guard let args = call.arguments as? Dictionary<String, String>,
+                                      let domain = args["domain"],
+                                      let ticker = args["ticker"] else {
+                                    result(nil)
+                                    return
+                                }
+                                
+                                guard let resolution = self?.resolution else {
+                                    result(nil)
+                                    return
+                                }
+                                        
+                                resolution.addr(domain: domain, ticker: ticker) { addrResult in
+                                  var address : String = ""
+                                    
+                                  switch addrResult {
+                                      case .success(let returnValue):
+                                        address = returnValue
+                                      case .failure(let error):
+                                        print("Expected Address, but got \(error)")
```

Comment:
Here you have different error catch with android side (` handler.post(() -> result.error("INVALID DOMAIN", e.getMessage(), null));` in android code). So need to send error in same way as on android side.